verify/Dockerfile

```diff
@@ -46,28 +46,41 @@ RUN dnf -y update && \
     dnf -y builddep /tmp/cockpit.spec && \
     dnf clean all
 
-RUN mkdir -p /usr/local/bin /home/user /build
-ADD cockpit-verify vm-prep network-cockpit.xml /usr/local/bin/
+RUN mkdir -p /usr/local/bin /home/user /build /build/tmp /secrets /images
+ADD cockpit-verify install-service vm-prep network-cockpit.xml /usr/local/bin/
 
-RUN chown -R user /build /home/user
 RUN usermod -a -G mock user
 RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
+RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
+    ln -s /images /build/images && \
+    mkdir -p /home/user/.config /home/user/.ssh && \
+    ln -snf /secrets/ssh-config /home/user/.ssh/config && \
+    ln -snf /secrets/github-token /home/user/.config/github-token && \
```

Comment:
Yes, of course - hence my thought of copying them at container startup (not in the Dockerfile), then they can be chowned and chmodded around as needed.

A nice goal would be to support a read-only mount of a user's ~/.ssh directory and copy/chown those into the container's `/run`, to completely avoid any on-disk secret copies. If that's not (easily) possible yet, and the secrets need to be  copied to `/var` or `/opt` on the host, then this approach is fine for now.